core/ui/compose/designsystem/src/test/kotlin/app/k9mail/core/ui/compose/designsystem/atom/textfield/PasswordTextFieldOutlinedKtTest.kt

```diff
@@ -0,0 +1,165 @@
+package app.k9mail.core.ui.compose.designsystem.atom.textfield
+
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.assertIsEnabled
+import androidx.compose.ui.test.assertIsNotEnabled
+import androidx.compose.ui.test.onNodeWithContentDescription
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.test.onNodeWithText
+import androidx.compose.ui.test.performClick
+import androidx.compose.ui.test.performTextInput
+import app.k9mail.core.ui.compose.designsystem.R
+import app.k9mail.core.ui.compose.testing.ComposeTest
+import assertk.assertThat
+import assertk.assertions.isEqualTo
+import org.junit.Test
+
+class PasswordTextFieldOutlinedKtTest : ComposeTest() {
+
+    @Test
+    fun `should toggle password visibility`() = runComposeTest {
+        setContent {
+            PasswordTextFieldOutlined(
+                value = VALUE,
+                onValueChange = ON_VALUE_CHANGE,
```

Comment:
The name of the constant doesn't give a hint as to what the function does. Maybe it should be named something like `IGNORE_VALUE_CHANGE`. But in this particular case, going with `onValueChange = {}` would probably be the option that is the easiest to read.

Reply:
Hmm, at some point the IDE required me to add `(String) -> Unit` as type and that's why this is even here. But now it also works with only `onValueChange = {}`. Strange